### PR TITLE
Docker for services

### PR DESCRIPTION
**Pull request by SaintAngeLs**
Update the dockerfiles for the services

### CODE REVIEW

File: SwiftParcel/d-docker-compose/micro-services-local.yml [hunk 1]
```diff
@@ -2,12 +2,23 @@ version: "2.1"
 
 services:
 
+ api-gateway:
+    build: ../../SwifttParcel.API.Gateway
+    container_name: api-gateway
+    restart: unless-stopped
+    environment:
+      - NTRADA_CONFIG=ntrada.docker # TODO: Add this file
+    ports:
```
Comment (eggwhat) [inline SwiftParcel/d-docker-compose/micro-services-local.yml:11]: There is a typo in build path.

File: SwiftParcel/d-docker-compose/micro-services-local.yml [hunk 2]
```diff
@@ -2,12 +2,23 @@ version: "2.1"
...
+      - 5000:80
+    networks:
+      - pacco
+
   identity-service:
     build: ../../SwifttParcel.Services.Identity
     container_name: apa/identity-service
     restart: unless-stopped
```
Comment (eggwhat) [inline SwiftParcel/d-docker-compose/micro-services-local.yml:19]: Idk why some container names have this apa prefix and some don't. Also my docker doesn't like slashes in name.

**Review by eggwhat — APPROVED** [review]
Seems to be mostly working but we should probably dockerize remaining services like rabbitmq

File: SwiftParcel/d-docker-compose/micro-services-local.yml (hunk 1 above)
Comment (SaintAngeLs) [reply SwiftParcel/d-docker-compose/micro-services-local.yml:11]: I agree

File: SwiftParcel/d-docker-compose/micro-services-local.yml (hunk 2 above)
Comment (SaintAngeLs) [reply SwiftParcel/d-docker-compose/micro-services-local.yml:19]: apa is hypothetical name of the dockerhub repo owner. The micro-servirces-local are to build the existing Dockerfiles. Services  are to build it from the  images from the dockerhub.

**Pull request merged**